test/navigation/nonsipp/LoansMadeOrOutstandingNavigatorSpec.scala

```diff
@@ -317,7 +317,7 @@ class LoansMadeOrOutstandingNavigatorSpec extends BaseSpec with NavigatorBehavio
             srn => LoansListPage(srn, true),
             (srn, _) =>
               controllers.nonsipp.common.routes.IdentityTypeController
-                .onPageLoad(srn, refineMV(3), NormalMode, IdentitySubject.LoanRecipient),
+                .onPageLoad(srn, refineMV(1), NormalMode, IdentitySubject.LoanRecipient),
             srn =>
```

Comment:
Do we need similar test to this one for the other journeys?

Reply:
Added missing tests